templates/speaker/show.html.twig

```diff
@@ -31,9 +31,7 @@
                 </tr>
                 <tr>
                     <th>Photo</th>
-                    <td> 
-                        <img src="{{ speaker.photo|storage_url }}" alt="Avatar" class="img-fluid">
-                    </td>
+                    <td><img src="{{ speaker.photo|storage_url }}" alt="Avatar" class="img-fluid"></td>
```

Comment:
As @tdutrion already said, you don't have to modify this file (as your modification is just to change the format of these 3 lines). So revert your changes.